.functional-tests.env

```diff
@@ -30,3 +30,4 @@ EQ_SUBMISSION_CONFIRMATION_BACKEND=log
 VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS=35
 SDS_API_BASE_URL=http://localhost:5003
 OIDC_TOKEN_BACKEND=local
+EQ_MAX_INSTANCES=1
```

Comment:
This is vague and makes me think it controls runner's no. of instances. I would rename to something like `EQ_FUNCTIONAL_TEST_MAX_INSTANCES`.

Reply:
@MebinAbraham I had to slightly change it now so it's a bool and ternary used to get its value, but at least it seems like all flakiness is gone. Maybe we should rename this var now?